app/gateway/changelog.md

```diff
@@ -5,12 +5,301 @@ no_version: true
 ---
 
 ## 2.7.0.0
-**Release Date:** TBD
+**Release Date:** 2021/12/16
 
-#### Deprecated
+### Features
+
+#### Enterprise
+* Kong Gateway now supports installations on [Debian 10 and 11](/gateway/2.7.x/install-and-run/debian/).
+* This release introduces consumer groups, a new entity that lets you
+manage custom rate limiting configuration for any defined subsets of consumers.
+To use consumer groups for rate limiting, configure the [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced)
+plugin with the `enforce_consumer_groups` and `consumer_groups` parameters,
+and use the `/consumer_groups` endpoint to manage the groups.
+  * [Consumer groups reference](/gateway/2.7.x/admin-api/consumer-groups/reference)
+  * [Consumer groups examples](/gateway/2.7.x/admin-api/consumer-groups/examples)
+* The data plane configuration cache can now be encrypted or disabled. Two new
+configuration options have been added:
+  * `data_plane_config_cache_mode`: The cache can be `unencrypted`, `encrypted`, or `off`.
+  * `data_plane_config_cache_path`: If unencrypted, the config cache is stored
+  by default in `config.cache.json.gz`. You can use this setting to specify a
+  custom path for the cache.
+* The `/license/report` API endpoint now provides
+[monthly throughput usage reports](/gateway/2.7.x/plan-and-deploy/licenses/report).
+
+#### Dev Portal
+* The Dev Portal API now supports `sort_by={attribute}` and `sort_desc`
+query parameters for sorted list results.
+* At Dev Portal account and admin account creation, you can now use the `email` of
+the `consumer_claim` to either match with an existing consumer or create a
+new account.
```

Comment:
```suggestion
* For Dev Portals with the [OpenID Connect](/hub/kong-inc/openid-connect) plugin enabled, login is successful on the first try if the credentials provided are valid and stored in IDP. If the email does not match existing data associated with the account, a new account is automatically created.
```

The current text requires knowledge of the codebase. I updated to what the user can expect from this change.